git-cherry: add page (#4473)
tldr-pages/tldr: pages/common/git-cherry.md

# git cherry

> > Find commits yet to be applied to upstream.
> > Commits prefixed with - have an equivalent; those with + do not.

Reviewer: Perhaps this will read better?
```suggestion
> Find commits that have yet to be applied upstream.
```

Author: Regarding line three, I just took the blurb from git's docs, but I'd agree that "that have" adds clarity.

As for line four, I was hoping to provide a little context because it's frankly confusing, but I suppose that's just once there's any level of complexity, so it'd be fine to leave out.

> More information: <https://git-scm.com/docs/git-cherry>.

> - Show commits and their message with an equivalent in upstream:

Reviewer: Perhaps this will read better?
```suggestion
- Show commits and their messages with the equivalent commit upstream:
```

Author: @sbrl `with the equivalent commit upstream` sounds to me like it's suggesting it shows a list of "commit + message" and "upstream commit" pairs, which it does not.  How about 

```suggestion
- Show commits (and their messages) with equivalent commits upstream:
```

Reviewer: Ah, I see - I must has misunderstood. That looks better - let's go with that.
